projects/js-packages/babel-plugin-replace-textdomain/CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.0.12] - 2022-07-22
+
```

Comment:
A quick suggestion here is to make sure there aren't any blank changelog entries - in cases like this it's generally just package dependency updates so you can copy-paste the previous entry:
```suggestion
## [1.0.12] - 2022-07-22
### Changed
- Updated package dependencies.
```
There are a number of other packages with blank entries too.